Packs/GoogleMaps/Integrations/GoogleMaps/GoogleMaps.py

```diff
@@ -0,0 +1,126 @@
+from CommonServerPython import *  # noqa # pylint: disable=unused-wildcard-import
+from CommonServerUserPython import *  # noqa
+
+# Disable insecure warnings
+OUTPUTS_PREFIX = 'GoogleMaps'
+NO_RESULTS_MESSAGE = 'No matching places were found.'
+
+requests.packages.urllib3.disable_warnings()  # pylint: disable=no-member
+
+
+class Client(BaseClient):
+    def __init__(self, api_key: str, base_url: str, proxy: bool, insecure: bool):
+        """
+        Client to use in the IPinfo integration. Uses BaseClient
+        """
+        super().__init__(base_url=base_url,
+                         verify=not insecure,
+                         proxy=proxy)
+        self.api_key = api_key
+
+    def google_maps_geocode(self, search_address: str, error_on_no_results: bool) -> List[CommandResults]:
+        # noinspection PyTypeChecker
+        response: Dict[str, Any] = self._http_request(method='GET', url_suffix='geocode/json?',
+                                                      params=assign_params(address=search_address, key=self.api_key))
+
+        status = demisto.get(response, 'status')
+        if status == 'OK':
+            return parse_response(response, search_address)
```

Comment:
This should be called from the `_command` function.

Reply:
@dorschw please address this point.

Reply:
done on https://github.com/demisto/content/pull/12934/commits/8ec77725f2dd4c085ad199892c283d7565e3b085